pkg/server/server.go

```
@@ -0,0 +1,66 @@
+package server
+
+import (
+	"context"
+	"log"
+	"net"
+	"tuber/pkg/k8s"
+	"tuber/pkg/proto"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+)
+
+// Serve serves
+func Serve() {
+	lis, err := net.Listen("tcp", ":9000")
+
+	if err != nil {
+		log.Fatalf("failed to listen: %v", err)
```

this func doesnt block until `grpc.Serve`, so this should probably return an error and panic out in `start.go`